data/srfi-data.scm

```diff
@@ -2171,12 +2171,13 @@
  (keywords data-structure)
  (draft-date "2023-11-14"))
 ((number 251)
- (status draft)
+ (status final)
```

Comment:
Changes to this file should not be part of this change list.

Also, these changes reflect the current status of the file, so I'm not sure why they're here, anyway.